### PR TITLE
SISRP-13260 - Build "Students" card in delegate user's default view ("Toolbox")

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-13260

This PR adds a new SVG file and some text changes from @rachel, and a suggestion from @christianvuerings re the `$scope.students` setting logic

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/delegateStudentsController.js [hunk 1]
```diff
@@ -37,12 +37,21 @@ angular.module('calcentral.controllers').controller('DelegateStudentsController'
...
     }
+
+    /**
+     * If at least one student grants only phone privilege, this flag lets us
+     * show the global 'phone-and-in-person privileges' explanatory paragraph in
+     * the template.
+     */
+    if (!viewable && !$scope.showPhoneInPersonPrivilegesMessage && student.privileges[phone]) {
+      $scope.showPhoneInPersonPrivilegesMessage = true;
+    }
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/delegateStudentsController.js]: @dfkaye I think this should be a one-line expression such that showPhoneInPersonPrivilegesMessage is always assigned. `$scope.showPhoneInPersonPrivilegesMessage = $scope.showPhoneInPersonPrivilegesMessage || student.privileges[phone]` (?)

File: src/assets/javascripts/angular/controllers/widgets/delegateStudentsController.js [hunk 2]
```diff
@@ -37,12 +37,21 @@ angular.module('calcentral.controllers').controller('DelegateStudentsController'
     student.delegateAccess = viewable;
 
     /**
-     * If at least one student grants no privileges, this flag let's us show the
+     * If at least one student grants no privileges, this flag lets us show the
      * global 'No privileges' explanatory paragraph in the template.
      */
     if (!viewable && !$scope.showNoPrivilegesMessage && !student.privileges[phone]) {
       $scope.showNoPrivilegesMessage = true;
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/delegateStudentsController.js]: @dfkaye Ditto, as below. Can this be a one-liner?

**Comment by dfkaye** [comment]
@johncrossman ~ one-liners are in ~ https://github.com/dfkaye/calcentral/commit/1d5f081096ace155a504a68b2eb276c7486156ef

File: src/assets/templates/widgets/toolbox/delegate_students.html [hunk 3]
```diff
@@ -34,12 +34,19 @@
       </ul>
       <hr>
       <div>
+        <p data-ng-if="showPhoneInPersonPrivilegesMessage">
+          <!-- using the p tag for bottom padding/margin which divs don't have -->
```
Comment (johncrossman) [inline src/assets/templates/widgets/toolbox/delegate_students.html]: @dfkaye Not a blocker but you don't need to explain design decisions in such comments.

**Comment by johncrossman** [comment]
Thanks @dfkaye

File: src/assets/images/svg/photo_unavailable_bear_72x96.svg [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (christianvuerings) [inline src/assets/images/svg/photo_unavailable_bear_72x96.svg]: @dfkaye could we run this through `svgo` to optimize this file?

File: src/assets/templates/widgets/toolbox/delegate_students.html (hunk 3 above)
Comment (christianvuerings) [inline src/assets/templates/widgets/toolbox/delegate_students.html]: @dfkaye indeed, please remove this

Comment (dfkaye) [inline src/assets/templates/widgets/toolbox/delegate_students.html]: @christianvuerings I'll push this change now...

**Pull request merged**